api/src/services/config/config.ts

```diff
@@ -1,5 +1,36 @@
+import type { Query } from 'types/graphql'
+
 import { getUserProjectConfig } from 'src/util/project'
 
-export const studioConfig = async () => {
-  return getUserProjectConfig().studio
+export const studioConfig = async (): Promise<Query['studioConfig']> => {
+  const config = await getUserProjectConfig()
+
+  return {
+    id: 'sc',
```

Comment:
Is there a benefit of having a really short `id` here? Or could we make it longer to make it more descriptive? Like `'studioConfig'`

Reply:
The benefit of having an id here is that Apollo Client needs both an id and typename in order to cache the value. So,  I added ids so the client could cache and update the cache properly -- which was important when changing the ssd enabled flag in the test project while studio was still running. Otherwise if didn't necessarily show the change all the time.

Am making more friendly. I could see these be needed maybe when having multiple team projects to append the project id to each config to ensure the config values are unique and cache properly.